test/retry.js

```diff
@@ -247,3 +249,13 @@ test('works when defaults.options.retry is not an object', async t => {
 	});
 	t.is(retryCount, 0);
 });
+
+test('retry function can throw', async t => {
+	await t.throws(got(`${s.url}/413`, {
+		retry: {
+			retries: () => {
+				throw new Error('Simple error');
+			}
+		}
+	}), 'Simple error');
```

Comment:
It's a good practice to put the message in a variable and use it in both to make absolutely sure they are the same and stay the same in the future too.

Reply:
Yeah.